build.rs

```diff
@@ -637,7 +640,7 @@ fn main() -> Result<()> {
         // TODO: Find out how we can set -undefined dynamic_lookup here (if this is possible)
     }
 
-    let env_vars = ["LD_LIBRARY_PATH", "PATH", "PYTHON_SYS_EXECUTABLE", "LIB"];
+    let env_vars = ["LD_LIBRARY_PATH", "PATH", "PYO3_INTERPRETER", "LIB"];
```

Comment:
Is this compatible with old tools?

Reply:
Good catch, we'll need both variables in here for now!